File: C++/monotonicarray.cpp
using namespace std;

bool isMonotonic(vector<int> array) {
  // Write your code here.
	bool increase = true;
	bool decrease = true;
	for(int i=1;i<array.size();++i){
		if(array[i]<array[i-1])
			decrease = false;
		if(array[i]>array[i-1])
			increase = false;
	}
  return increase||decrease;
}

File: Python/monotonicarray.py
def isMonotonic(array):
    # Write your code here.
	increase = True
	decrease = True
	for i in range(1, len(array)):
		if array[i]<array[i-1] : decrease = False
		if array[i]>array[i-1] : increase = False
	return increase or decrease		
    pass
